lib/yt/request.rb

```diff
@@ -196,7 +196,7 @@ def parse_response!
     # - when the user has reached the quota for requests/second, and waiting
     #   for a couple of seconds might solve the connection issues.
     def run_again?
-      refresh_token_and_retry? ||
+      refresh_token_and_retry? && sleep_and_retry?(3) ||
```

Comment:
I'd like to make the retry calls iterative, but that's an improvement we can make later.